lib/spack/spack/build_systems/autotools.py

```diff
@@ -184,12 +189,6 @@ def patch(self):
             ``config.guess``
         """
```

Comment:
This will solve my other confusion where it kept saying we patched the package even though I didn't define `patch`.